R/database-snomed.R

```diff
@@ -0,0 +1,536 @@
+#' Add SNOMED tables
+#'
+#' Creates or updates a SNOMED CT database
+#'
+#' @param path_destination Character string. Path where the database files
+#'   will be written.
+#' @param release_index Integer. Which TRUD release index to use (1 = latest).
+#'
+#' @return Invisible `TRUE` on success
+#'
```

Comment:
```suggestion
#' @export
```

Needs to be exported. Some `@examples` would also be useful (and needed to pass the checks).